infrastructure/lib/queue-permissions.ts

```diff
@@ -0,0 +1,7 @@
+/**
+ * Supported Queue Permissions for the application.
+ */
+export enum QueuePermissions {
+  SEND = "SEND",
+  CONSUME = "CONSUME",
+}
```

Comment:
Why not just make a file `infrastructure/lib/permissions.ts`? Why do queue and tables need their own files? Can we define a more general `CREATE`, `READ`, `UPDATE`, `DELETE` set of values that can be used across resources?